guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -54,13 +44,13 @@ When the `/var/lib/pulp` directory is mounted using an NFS share, SELinux blocks
 To avoid this, specify the SELinux context of the `/var/lib/pulp` directory in the file system table by adding the following lines to `/etc/fstab`:
 
 ----
-nfs.example.com:/nfsshare  /var/lib/pulp/content  nfs  context="system_u:object_r:httpd_sys_rw_content_t:s0"  1 2
+nfs.example.com:/nfsshare  /var/lib/pulp  nfs  context="system_u:object_r:var_lib_t:s0"  1 2
 ----
 
 If NFS share is already mounted, remount it using the above configuration and enter the following command:
 
 ----
-# chcon -R system_u:object_r:httpd_sys_rw_content_t:s0 /var/lib/pulp
+# chcon -R system_u:object_r:var_lib_t:s0 /var/lib/pulp
```

Comment:
This is an instruction that breaks things since the subdirectories have different contexts:
```suggestion
# restorecon -R /var/lib/pulp
```

Reply:
I commented below about the mixutre of labels, but @ekohl I think your idea to use restorecon to ensure they are all correct is right. That will fix this.